src/Component.php

```diff
@@ -137,16 +143,28 @@ private function registerAssets()
 
     public function captureMessage($message, $params, $levelOrOptions = [], $stack = false, $vars = null)
     {
+        if (!$this->enabled) {
+            Yii::trace(Json::encode([$message, $params, $levelOrOptions]), __CLASS__);
+            return '-1';
```

Comment:
captureMessage normally returns the event id as a string, but it can also return null according to its doc.

We should also return null if the component is not enabled.